File: app/src/main/java/com/sierra/vagabond/main/details/DetailsPresenterImpl.java
package com.sierra.vagabond.main.details;

import java.util.List;

public class DetailsPresenterImpl implements DetailsContract.Presenter {

  public DetailsContract.View view;

  public DetailsPresenterImpl(DetailsContract.View view) {
    this.view = view;
  }

  @Override
  public void loadImages(List<String> imageUrls) {
    view.setDataToRecyclerView(imageUrls);
  }

  @Override
  public void loadAreaDescription(String recAreaName) {}

  @Override
  public void areaDirections(String recAreaName) {}
}

File: app/src/main/java/com/sierra/vagabond/main/details/DetailsPresenterImpl.kt
package com.sierra.vagabond.main.details

class DetailsPresenterImpl(var view: DetailsMvp.View) : DetailsMvp.Presenter {

    override fun loadImages(imageUrls: List<String>) {
        view.setDataToRecyclerView(imageUrls)
    }

    override fun loadAreaDescription(recAreaName: String) {}

    override fun areaDirections(recAreaName: String) {}
}
